docker/README.md

```diff
@@ -0,0 +1,89 @@
+# NSoT Docker Image
+
+This Docker image runs NSoT. Perfect for quick developing and even deploying in
+production.
+
+## Using this image
+
+Image tags should correspond with NSoT release version numbers. Basic usage is
+like:
+
+```bash
+
+$ SECRET='YkV0Q0lYT3RpN0ppMGZ2Q0tMVE8wbW55WjQ0QkFpdlBvUWpLbFpBTlVZSW1DVnZmMWFNTWpteDFGVDFKUXg2'
+$ docker run -p -d --name=nsot -e NSOT_SECRET=$SECRET dropbox/nsot:0.15.4
```

Comment:
Whoops, yeah this should be `-p 8990:8990`. `-P` will assign random host port -> container exposed ports